### PR TITLE
Models for users

### PR DESCRIPTION
**Pull request by sayantan300**
As asked by @MattA-Official for testing my work

### CODE REVIEW

File: models/users.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  if((this.avatar=="1")||(this.avatar=="2")||(this.avatar=="3")||(this.avatar=="4")){ return `https://cdn.discordapp.com/embed/avatars/${this.avatar}.png`;}
+  else{
+ var ani=false;
+ if(this.avatar.startsWith("a_")) ani=true;
+ const aniurl=`https://cdn.discordapp.com/avatars/${this.id}/${this.avatar}.gif`;
+ const nonurl=`https://cdn.discordapp.com/avatars/${this.id}/${this.avatar}.png`;
+ const url = (ani)?aniurl:nonurl;
+ return url;
+  }
+});
```
Comment (MattA-Official) [inline models/users.ts:40]:
```suggestion
Users.virtual('avatarURL').get( () => {
  if ( (this.avatar === "1") || (this.avatar==="2") || (this.avatar=="3") || (this.avatar === "4") ) return `https://cdn.discordapp.com/embed/avatars/${this.avatar}.png`;
  else {
    const ani = this.avatar.startsWith("a_") ? true : false;
    const aniurl = `https://cdn.discordapp.com/avatars/${this.id}/${this.avatar}.gif`;
    const nonurl = `https://cdn.discordapp.com/avatars/${this.id}/${this.avatar}.png`;
    const url = ani ? aniurl : nonurl;
    return url;
  };
});
```

File: models/users.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+Users.virtual('tag').get(function(){
+ return `${this.username}#${this.discriminator}`;
+})
```
Comment (MattA-Official) [inline models/users.ts:43]:
```suggestion
Users.virtual('tag').get( () => {
 return `${this.username}#${this.discriminator}`;
})
```

File: models/users.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+var users;
```
Comment (MattA-Official) [inline models/users.ts:45]:
```suggestion
let users;
```

**Review by MattA-Official — CHANGES_REQUESTED** [review]

File: models/users.ts (hunk 1 above)
Comment (sayantan300) [reply models/users.ts]: Hey, you cannot bind `this` with arrow functions

**Review by MattA-Official — APPROVED** [review]

**Pull request merged**